exporters/prometheus/src/main/java/io/opentelemetry/exporter/prometheus/Otel2PrometheusConverter.java

```diff
@@ -484,6 +486,16 @@ private Labels convertAttributes(
       if (scope.getVersion() != null) {
         labelNameToValue.putIfAbsent(OTEL_SCOPE_VERSION, scope.getVersion());
       }
+      String schemaUrl = scope.getSchemaUrl();
+      if (schemaUrl != null) {
+        labelNameToValue.putIfAbsent(OTEL_SCOPE_SCHEMA_URL, schemaUrl);
```

Comment:
The preference it to add it even when it is empty.
See: https://github.com/open-telemetry/opentelemetry-go/pull/5947#issuecomment-2873152108